lib/rpush/client/active_model/apns2/notification.rb

```diff
@@ -3,6 +3,20 @@ module Client
     module ActiveModel
       module Apns2
         include Rpush::Client::ActiveModel::Apns
+
+        module Notification
+          MAX_PAYLOAD_BYTESIZE = 4096
```

Comment:
This change is key to squashing the behavior described in https://github.com/rpush/rpush/issues/545, and I'd like to provide some additional context to make reviewing a bit easier for other reviewers:

Prior to this PR, the APNS2 notification payload was incorrectly marked as invalid on notifications where the payload byte size was _greater than_ 2kb, but _less than_ 4kb. In other words, Apple has a [size limit of 4kb for APNS2](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CreatingtheNotificationPayload.html#:~:text=The%20maximum%20size%20of%20the,size%20is%205KB%20(5120%20bytes)), but rpush would mark any notifications with a payload greater than 2kb as invalid.